jishaku/features/management.py

```diff
@@ -241,7 +241,7 @@ async def jsk_sync(self, ctx: ContextA, *targets: str):
             )
             translator = getattr(self.bot.tree, 'translator', None)
             if translator:
-                payload = [await command.get_translated_payload(translator) for command in slash_commands]
+                payload = [await command.get_translated_payload(self.bot.tree, translator) for command in slash_commands]
             else:
                 payload = [command.to_dict() for command in slash_commands]
```

Comment:
The `to_dict()` call on line 244 also needs to be passed the tree as an argument.